exchanges/gateio/gateio.go

```diff
@@ -3753,8 +3753,12 @@ func (g *Gateio) IsValidPairString(currencyPair string) bool {
 	if len(currencyPair) < 3 {
 		return false
 	}
-	if strings.Contains(currencyPair, g.CurrencyPairs.RequestFormat.Delimiter) {
-		result := strings.Split(currencyPair, g.CurrencyPairs.RequestFormat.Delimiter)
+	pf, err := g.CurrencyPairs.GetFormat(asset.Spot, true)
+	if err != nil {
+		return false
```

Comment:
I am just pointing out that something ungodly is going down if this is erroring, so just a _suggestion_ to return an error along with the bool. But up to you.

Reply:
Thought about it.
But I think that shouldn't be the responsibility of this function, and I want to keep `IsXYZ` functions slim.
According to the data we have, this is definitely *not* a valid string 😭 😂